### ISSUE TITLE
Federate JSON config parameters have mixed style

### ISSUE THREAD

**Issue by trevorhardy** [GMLC-TDC/HELICS-src#477]
Looking at ....examples/example_files/example_value_fed.json (among many other places), the JSON config. file has mixed camelCase and snake_case and shouldn't:

```
"only_update_on_change":false, 
"only_transmit_on_change":false, 
"source_only":false, 
...
"timeDelta":0.0, 
"outputDelay":0, 
"inputDelay":0,
...
```

**Comment by afisher1** [comment]
Thanks for pointing this out @trevorhardy . @phlptp this is coding style issue rather than anything else. I personally don't like mixed variable nameing styles especially on class members. So far the only culprit I can find is the CoreFederateInfo class. Fixing this would be a breaking change but since we've have incompatibility between 2.0 and 1.x anyways I'd say this is worth fixing. Opinions?

**Comment by afisher1** [comment]
Strike that. I don't see the mixed case in 2.0 branch for CoreFederateInfo so that's good. It appears that we are supposed to be using CamelCase for everything however I do see only_update_on_change and only_transmit_on_change in the example_message_fed.json configuration file but I can't find which src file has those properties.

**Comment by phlptp** [comment]
The general formatting rule is that variables and member variables are camelCase.  snake_case is used for constants and flags to distinguish them.  Classes are CamelCase, with one exception those are that typedefs or wrappers for fundamental types  (void *),  int32_t, enumerations.    I am not entirely sure this is perfectly followed everything in the code base but when I see something not matching I usually fix it if possible.    That being said there are probably some inconsistencies that didn't get fixed in 2.0 yet for the configuration files.   I will take a look at that at some point soon.
